src/configs/os.rs

```diff
@@ -0,0 +1,77 @@
+use indexmap::{indexmap, IndexMap};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(feature = "config-schema", derive(schemars::JsonSchema))]
+#[serde(default)]
+pub struct OSConfig<'a> {
+    pub format: &'a str,
+    pub style: &'a str,
+    pub symbols: IndexMap<String, &'a str>,
+    pub disabled: bool,
+}
+
+impl<'a> OSConfig<'a> {
+    pub fn get_symbol(&self, key: &str) -> Option<&'a str> {
+        self.symbols.get(key).cloned()
```

Comment:
Looking at it this again, I think I would prefer if this table would work case-insensitively. But we can also change this later in case the case-sensitive version ends up confusing users.

Reply:
Working on it!